captum/attr/_core/deep_lift.py

```diff
@@ -77,14 +86,15 @@ def __init__(self, model: Module) -> None:
     def attribute(
         self,
         inputs: TensorOrTupleOfTensors,
-        baselines: Union[
-            Tensor, int, float, Tuple[Union[Tensor, int, float], ...]
+        baselines: Optional[
+            Union[Tensor, int, float, Tuple[Union[Tensor, int, float], ...]]
         ] = None,
         target: Optional[
-            Union[int, Tuple[int, ...], Tensor, List[Tuple[int, ...]]]
+            Union[int, Tuple[int, ...], Tensor, List[Tuple[int, ...]], List[int]]
         ] = None,
         additional_forward_args: Any = None,
-        custom_attribution_func: Callable[..., Tuple[Tensor, ...]] = None,
+        return_convergence_delta: "Literal"[False] = False,
```

Comment:
Literal is included as a string since it is imported only if type checking, and would result in runtime errors if used directly.